lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -498,6 +498,77 @@ defmodule ExUnitTest do
     ExUnit.configure(seed: global_seed)
   end
 
+  @doc """
+    Skipped and excluded tests should be included in the stats as well as printed to stdout.
+    On the other hand, invalid tests should be marked as failures in the stats,
+    but still be printed as "invalid" to stdout.
+
+    If setup_all fails, the skipped and excluded tests should not be counted as invalid or failures.
+  """
+  test "skipped and excluded" do
```

Comment:
We already have tests for skipped i this module, I would propose to rename this "accounts skipped and excluded on setup_all errors.

Reply:
Or, even better, this file has no test for "setup_all", so maybe we should call it "setup_all has invalid, skipped and excluded tests". :)

Reply:
how about "setup_all fails and module has skipped and excluded tests", since its it not setup_all that has invalid test but the cause of them being rendered as invalid